src/main/kotlin/convergence/discord/frat/FratCommands.kt

```diff
@@ -33,11 +33,40 @@ val englishToGreek = mapOf(
     'Y' to 'Ψ',
 )
 
-fun brotherInfo(args: List<String>, searchCriteria: (BrotherInfo) -> String?): DiscordOutgoingMessage {
-    val name = args.joinToString(" ").lowercase()
-    val info = brotherInfo.firstOrNull { searchCriteria(it)?.lowercase() == name }
+fun getBrotherInfo(args: String, searchCriteria: (BrotherInfo) -> String?): BrotherInfo {
+    return brotherInfo.firstOrNull { searchCriteria(it)?.lowercase() == name }
         ?: brotherInfo.firstOrNull { searchCriteria(it)?.lowercase()?.startsWith(name) == true }
         ?: brotherInfo.firstOrNull { searchCriteria(it)?.lowercase()?.contains(name) == true }
+}
+
+fun brotherLineRec(name: String, searchCriteria: (BrotherInfo) -> String?, depth: Int = 10): List<String>  {
+    val info = getBrotherInfo(name, searchCriteria)
+        ?: return List<String>()
+
+    if (depth == 0) {
+        return List<String>()
+    }
+    
+    return brotherLineRec(info.bigBrother.ifBlank { "N/A" }, searchCriteria, depth-1).add(info.bigBrother)
+}
+
+fun brotherLine(args: List<String>, searchCriteria: (BrotherInfo) -> String?): DiscordOutgoingMessage  {
+    val name = args.joinToString(" ").lowercase()
+    val startInfo = getBrotherInfo(name, searchCriteria)
+        ?: return DiscordOutgoingMessage("No brothers found searching for \"$name\".")
+    
+    val lineList = brotherLineRec(startInfo.firstName + " " + startInfo.lastName)
+
+    return DiscordOutgoingMessage(MessageCreateBuilder()
+            .addEmbeds(EmbedBuilder()
+                    .setTitle("Line for Brother #${startInfo.rosterNumber} ${startInfo.firstName} ${startInfo.lastName}")
+                    .addField("Line", lineList.joinToString("/r/n") true).build()
```

Comment:
Should probably just be \n (definitely not forward slash), missing a comma after the joinToString, the "true" just says whether the field is inline or not